src/app/payments/send-intraledger.ts

```diff
@@ -281,28 +279,24 @@ const executePaymentViaIntraledger = async ({
 }
 
 const addContactsAfterSend = async ({
-  senderAccountId,
-  senderUsername,
-  recipientAccountId,
-  recipientUsername,
+  senderAccount,
+  recipientAccount,
 }: {
-  senderAccountId: AccountId
-  senderUsername: Username | undefined
-  recipientAccountId: AccountId
-  recipientUsername: Username | undefined
+  senderAccount: Account
+  recipientAccount: Account
 }): Promise<true | ApplicationError> => {
-  if (recipientUsername) {
+  if (senderAccount.contactEnabled && recipientAccount.username) {
```

Comment:
I'm wondering it the use is more something like:

```
if (!(senderAccount.contactEnabled && recipientAccount.contactEnabled)) {
   return
}

continue...
```

ie: only add contact if both contact have contactEnabled

in the case of the quiz, I think it doens't make sense to have the funder add contacts, but also doesn't make sense to have the end user have the contact of the funder. 

wdyt?

Reply:
makes sense.. updated here https://github.com/GaloyMoney/galoy/pull/1696/commits/b35293a215a997ad35b6583d001bbbd8b5bfa8c5